formats/hmrc_manual/publisher/details.json

```diff
@@ -7,7 +7,7 @@
   ],
   "properties": {
     "body": {
-      "type": "string"
+      "$ref": "#/definitions/body"
```

Comment:
This feels mildly dangerous.  The frontend for rendering hmrc manuals doesn't understand that a body can be anything but a string.  If someone started submitting manuals with a multi-part govspeak+html body the schema would be accepted, but the frontend would break.  I don't know if other frontend apps would behave the same for bodies that used to be strings but now might be multi-part.